components/__tests__/locale-chooser.js

```diff
@@ -1,16 +1,18 @@
 import React from 'react'
-import {mount} from 'enzyme'
+import {render, mount} from 'enzyme'
 import toJson from 'enzyme-to-json'
 import {ThemeProvider} from 'glamorous'
 import {matcher, serializer} from 'jest-glamor-react'
 import GlobalStyles from '../../styles/global-styles'
 import LocaleChooser from '../locale-chooser'
 
+const {supportedLocales, fallbackLocale} = require('../../config.json')
+
 expect.addSnapshotSerializer(serializer)
 expect.extend(matcher)
 
 test('closed state', () => {
-  const wrapper = localeChooser()
+  const wrapper = renderLocaleChooser()
```

Comment:
I'm currently refactoring the tests at #185 and wondered if we really want to do the snapshot testing here.
Because imo  the rendering has so many lines of code that it isn't clear and it slows down the testing by some seconds.
I would suggest testing the rendering of the localization links (this test is must).
And if its needed we could also test some of the behaviors: e.g. the click event shows the list and sets the  `aria-expanded` to `true`, selected the first item on being opened, ...